frontend/packages/dev-console/src/components/health-checks/HealthChecks.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { PageHeading } from '@console/internal/components/utils';
+import FormSection from '../import/section/FormSection';
+import HealthCheckProbe from './HealthCheckProbe';
+import { HealthChecksProbeType } from './health-checks-types';
+
+interface HealthChecksProps {
+  pageTitle?: string;
+}
+
+const renderProbes = () => {
+  return (
+    <>
+      <HealthCheckProbe probe={HealthChecksProbeType.ReadinessProbe} />
+
+      <HealthCheckProbe probe={HealthChecksProbeType.LivenessProbe} />
+
+      <HealthCheckProbe probe={HealthChecksProbeType.StartupProbe} />
+    </>
+  );
+};
+
+const HealthChecks: React.FC<HealthChecksProps> = ({ pageTitle }) => {
+  return pageTitle ? (
+    <>
+      <PageHeading title={pageTitle} />
+      {renderProbes()}
+    </>
+  ) : (
+    <FormSection title="Health Checks">{renderProbes()}</FormSection>
+  );
```

Comment:
Based on my comments above you can simply do this -
```suggestion
const HealthChecksSection: React.FC<HealthChecksSectionProps> = ({ title }) => (
  <FormSection title={title}>
    <HealthCheckProbe probe={HealthChecksProbeType.ReadinessProbe} />
    <HealthCheckProbe probe={HealthChecksProbeType.LivenessProbe} />
    <HealthCheckProbe probe={HealthChecksProbeType.StartupProbe} />
  </FormSection>
);
```